Geometry/HGCalCommonData/interface/HGCalProperty.h

```diff
@@ -0,0 +1,20 @@
+#ifndef Geometry_HGCalCommonData_HGCalProperty_h
+#define Geometry_HGCalCommonData_HGCalProperty_h
+
+#include <cmath>
+#include <cstdint>
+
+class HGCalProperty {
```

Comment:
since all the functions are static and there are no member variables, this could just be a namespace instead of a class

Reply:
Done